### ISSUE TITLE
HS256 generates unexpected token

### ISSUE THREAD

**Issue by felipenoris** [GildedHonour/frank_jwt#31]
Header

```JSON
{
  "alg": "HS256",
  "typ": "JWT"
}
```

Payload

```JSON
{
  "userid": "CRISP",
  "group": "ATI",
  "resources": []
}
```

secret: "secret"

Should generate (using jwt.io): 
```
eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1c2VyaWQiOiJDUklTUCIsImdyb3VwIjoiQVRJIiwicmVzb3VyY2VzIjpbXX0.K9nfZnbMzF1-P1zXEQHeYYUz35NTbTPpT560wNG16DM
```

This lib returns:
```
eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJncm91cCI6IkFUSSIsInJlc291cmNlcyI6W10sInVzZXJpZCI6IkNSSVNQIn0=.XfroJa5E41zs03U1aPtP3Qy3Er4vIBVb0Be
tZ0GuBfk=
```

Even removing trailing `=` characters (#28) , this JWT is not valid.

**Comment by felipenoris** [comment]
Is this included in release v3.0.1 ?

**Comment by GildedHonour** [comment]
I don't know

It's included in 3.0.2
